com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/HTML5Bundler.java

```diff
@@ -442,6 +442,7 @@ public void bundleApplication(Project project, Platform platform, File bundleDir
                 FileUtils.copyFile(splashImage, new File(appDir, splashImage.getName()));
             }
         }
+        BundleHelper.moveBundleIfNeed(project, appDir);
```

Comment:
for html5 we put it right into the folder because it's a frequent case and makes it easier to upload the final bundle